.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,37 @@
+# Summary
+
+Please provide a summary of this issue.
+
+## If You Are Reporting a Bug
+
+If you're reporting a bug, please provide a detailed description of the problem.
+
+## If You Are Submitting a Feature Request
+
+If you're submitting a feature request, please provide as much details on what you'd like to see and 1 or more use cases. The more detail we have, the more quickly and efficiently we can discuss, decide and implement the feature.
+If this is a bug: provide a description of the problem
+
+## Everything Else
+
+If you're not submitting a bug or feature request, we still want to hear from you! Please feel free to write us and we'd love to have a conversation. As always, constructive conversation and details, where appropriate, are appreciated.
+
+# For Bugs: Replication Steps
```

Comment:
this might be better to place under the first header, "If you are reporting a bug"